website/content/partials/pki-double-migration-bug.mdx

```diff
@@ -0,0 +1,22 @@
+### PKI Storage Migration Revives Deleted Issuers
+
```

Comment:
```suggestion
- **Affected versions**: Vault versions before 1.11.1
- **Fixed in**: Vault 1.14.0, 1.13.4, 1.12.8, and 1.11.12
```

Avoid burying the lead ;) Moving the information about which versions are affected, makes it easier for readers to identify the most important piece of information.